include/chemfiles/Atom.hpp

```diff
@@ -66,6 +66,16 @@ class CHFL_EXPORT Atom {
     /// found.
     int atomic_number() const;
 
+    /// Try to get the Default Red(from RGB) element colors for viewers, if defined. Returns -1.0f if it can not be
+    /// found.
+    float color_r() const;
```

Comment:
I would rather have a single method returning the `(R, G, B)` vector at once. Something like `Vector3D rgb_color() const`.